kafka-consumer/src/main/resources/application.properties

```diff
@@ -25,3 +25,4 @@ mp.messaging.incoming.consumed.value.deserializer=org.apache.kafka.common.serial
 mp.messaging.incoming.consumed.enable.auto.commit=true
 mp.messaging.incoming.consumed.group.id=kafka-test-apps
 mp.messaging.incoming.consumed.auto.offset.reset=earliest
+mp.messaging.incoming.consumed.sasl.login.callback.handler.class=io.strimzi.kafka.oauth.client.JaasClientOauthLoginCallbackHandler
```

Comment:
I think this would break every other SASL authentication mechanism, or?

Reply:
Yeah. I was focused on the demo. Let's keep it commented for an easy use in oauth cases?